attributes/default.rb

```diff
@@ -63,6 +63,7 @@
 #
 # User for directories, configuration files and running Kafka.
 default.kafka.user = 'kafka'
+default.kafka.create_user = true
```

Comment:
I'd prefer if this was called `manage_user` or something along those lines. It would also be great if you could add a comment that briefly describes what the attribute does.